sql/analyzer/optimization_rules_test.go

```diff
@@ -264,33 +97,45 @@ func TestEvalFilter(t *testing.T) {
 		{
 			and(
 				eq(lit(5), lit(5)),
-				eq(col(0, "foo", "bar"), lit(5)),
+				eq(
```

Comment:
You could add a new `col` equivalent here to keep these concise

Reply:
yeah I think I sed replaced these, given the number of hours I spend fixing up these unit tests it would almost be better to rewrite these in a lisp IR or something